docs/source/models/engine_args.rst

```diff
@@ -0,0 +1,68 @@
+.. _engine_args:
+
+Engine Arguments
+==================
+
+Below, you can find an explanation of every engine argument for vLLM:
+
+* --model
+    Name or path of the huggingface model to use.
+* --tokenizer
+    Name or path of the huggingface tokenizer to use.
+* --revision
+    The specific model version to use. It can be a branch name, a tag name, or a commit id. If unspecified, will use the default version.
+* --tokenizer-revision
+    The specific tokenizer version to use. It can be a branch name, a tag name, or a commit id. If unspecified, will use the default version.
+* --tokenizer-mode
+    {auto,slow} The tokenizer mode.
+    
+    * -- "auto" will use the fast tokenizer if available
+    * -- "slow" will always use the slow tokenizer.
+* --trust-remote-code
+    Trust remote code from huggingface.
+* --download-dir
+    Directory to download and load the weights, default to the default cache dir of huggingface
+* --load-format
+    {auto,pt,safetensors,npcache,dummy} The format of the model weights to load. 
```

Comment:
Would you mind format the options so that it's more clear they are the options for the argument? Similar for other options.

Reply:
Should include formatting on dtype now